pkg/apis/app/v1alpha1/kogitoapp_types.go

```diff
@@ -131,6 +131,8 @@ type KogitoAppBuildObject struct {
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
 	// Internal Maven Mirror to be used during source-to-image builds to considerably increase build speed
 	MavenMirrorURL string `json:"mavenMirrorURL,omitempty"`
+	// Suppress the logs of downloading/uploading maven dependencies
```

Comment:
```suggestion
	// Show the logs of downloading/uploading maven dependencies
```

Reply:
wouldn't be better something like "switch to print or not dependencies download/upload status"

Reply:
set descriptio to `If set to true will print the logs for downloading/uploading of maven dependencies. Defaults to false`